index.rst

```diff
@@ -143,104 +164,207 @@ Core Components
     Resistance, components/sensor/resistance, omega.svg
-    Rotary Encoder, components/sensor/rotary_encoder, rotary_encoder.jpg
-    RuuviTag, components/sensor/ruuvitag, ruuvitag.jpg
-    SenseAir, components/sensor/senseair, senseair_s8.jpg
-    SDS011 Sensor, components/sensor/sds011, sds011.jpg
-    SDM Meter, components/sensor/sdm_meter, sdm220m.png
-    SCD30, components/sensor/scd30, scd30.jpg
-    SHTCx, components/sensor/shtcx, shtc3.jpg
-    SHT3X-D, components/sensor/sht3xd, sht3xd.jpg
-    SHT4X, components/sensor/sht4x, sht4x.jpg
-    SM300D2, components/sensor/sm300d2, sm300d2.jpg
-    SPS30, components/sensor/sps30, sps30.jpg
-    STS3X, components/sensor/sts3x, sts3x.jpg
-    SGP30, components/sensor/sgp30, sgp30.jpg
-    SGP40, components/sensor/sgp40, sgp40.jpg
-    TCS34725, components/sensor/tcs34725, tcs34725.jpg
-    Teleinfo, components/sensor/teleinfo, teleinfo.jpg
     Template Sensor, components/sensor/template, description.svg
-    TMP102, components/sensor/tmp102, tmp102.jpg
-    TMP117, components/sensor/tmp117, tmp117.jpg
-    TOF10120, components/sensor/tof10120, tof10120.jpg
-    Total Daily Energy, components/sensor/total_daily_energy, sigma.svg
-    TSL2561, components/sensor/tsl2561, tsl2561.jpg
-    Tuya Sensor, components/sensor/tuya, tuya.png
-    TX20, components/sensor/tx20, tx20.jpg
-    Ultrasonic Sensor, components/sensor/ultrasonic, ultrasonic.jpg
     Uptime Sensor, components/sensor/uptime, timer.svg
-    VL53L0x, components/sensor/vl53l0x, vl53l0x.png
     WiFi Signal Strength, components/sensor/wifi_signal, network-wifi.svg
-    Xiaomi BLE, components/sensor/xiaomi_ble, xiaomi_mijia_logo.jpg
+
+
+Air Quality / Gas
+*****************
+.. imgtable::
+
+    CCS811, components/sensor/ccs811, ccs811.jpg, CO2 & Volatile Organics
+    MH-Z19, components/sensor/mhz19, mhz19.jpg, CO2 & Temperature
+    HM3301, components/sensor/hm3301, hm3301.jpg, Particulate
+    PMSX003, components/sensor/pmsx003, pmsx003.svg, Particulate
+    SenseAir, components/sensor/senseair, senseair_s8.jpg, CO2
+    SDS011 Sensor, components/sensor/sds011, sds011.jpg, Particulate
+    SM300D2, components/sensor/sm300d2, sm300d2.jpg, Air Quality
+    SPS30, components/sensor/sps30, sps30.jpg, Particulate
+    STS3X, components/sensor/sts3x, sts3x.jpg, Temperature
+    SGP30, components/sensor/sgp30, sgp30.jpg, CO2 & Volatile Organic
+    SGP40, components/sensor/sgp40, sgp40.jpg, Volatile Organic
+    ZyAura, components/sensor/zyaura, zgm053.jpg, CO2 & Temp & Humidity
+
+
+Ambient Light / Colour
+**********************
+.. imgtable::
+
+    APDS9960, components/sensor/apds9960, apds9960.jpg, Colour & Gesture
+    BH1750, components/sensor/bh1750, bh1750.jpg, Lux
+    TCS34725, components/sensor/tcs34725, tcs34725.jpg, Lux & RGB Colour
+    TSL2561, components/sensor/tsl2561, tsl2561.jpg, Lux
+
+
+Analogue To Digital
+*******************
+.. imgtable::
+
+    ADC, components/sensor/adc, flash.svg, ESP internal
```

Comment:
This should be under both here and Core/Internal perhaps?

Reply:
I also wondered if I should add ADC to Core/Internal - happy to do so!

Reply:
done